src/common/src/FileUpload/FileUpload.stories.mdx

```diff
@@ -40,7 +40,7 @@ _The component below uses these options:_
 			cancelLabel="Annuleren"
 			fileUploadErrorLabel="dit bestand kan niet worden geüpload"
 			fileUploadInProgressLabel="wordt geupload"
-			options={{ noClick: true, noKeyboard: true, accept: 'image/png' }}
+			options={{ noClick: true, noKeyboard: true }}
```

Comment:
Should this not be configurable by prop if not with this sensible default?
I guess this determines whether a user can select one or multiple file types from using the dialog?

Reply:
I see this is just a storybook file now :speak_no_evil: